code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -370,7 +369,19 @@ obj/item/weapon/material/hatchet/tacknife/combatknife/fluff/katarina/handle_shie
 
 	icon_override = 'icons/vore/custom_clothes_vr.dmi'
 	item_state = "serdyhelm_mob"
-*/
```

Comment:
I'm not gonna mess with this file while you're working on it, but watch out for the mob_can_equip call on line 354 (in the section you're uncommenting) as per #711  - having it just return 1 and not call the parent function might lead to fucky behaviour.